ee/codegen/src/generators/workflow.ts

```diff
@@ -389,7 +390,7 @@ export class Workflow {
               this.workflowContext.getPortContextById(sourcePortId);
           } catch (e) {
             if (e instanceof BaseCodegenError) {
```

Comment:
[nit] Let's only warn on `NodePortGenerationError`, like how we made it more specific below